### DISCUSSION TITLE
how to reuse blog page (blog folder)'s style

### DISCUSSION THREAD

**Discussion by Zenuncl** [themefisher/bigspring-hugo-startup-theme#58] [Q&A]
I am trying to have two different folder for different type of blogs and trying to show both folder similar. I called the second folder event and trying to let `localhost:1313/event/` looks as same as `localhost:1313/blog/` how can I do that?

also if I want to use pricing style or other styles, do I have to put them in specific directory or I can do some tricks?

**Comment by tfsomrat** [comment]
Hello @SharkIng 

Thanks for using this theme, you can follow [this guide](https://docs.gethugothemes.com/guide/#maintain-page) to duplicate your pages.

**Reply by Zenuncl** [reply]
Thank you @tfsomrat that's awesome,

I checked the guide and I believe it's this part that can duplicate the blog listing page:
>duplicate an existing page is simple as creating a page. If you want to make a copy of privacy page, then you need to duplicate 2 files. The first one is content. Duplicate the privacy page folder from the content folder with _index.md file in it. Then change the folder name to your new page name (i.e. terms-conditions). Now you have your content ready, it’s time to duplicate the structure or the markup for this new page. Go to layouts folder and also duplicate the same page folder here, in our stage, it’s privacy page. And give the same name that you give the content folder (i.e. terms-conditions). Now you have successfully duplicate a page.

It says "Go to layouts folder and also duplicate the same page folder here" so which folder/file is the `/blog/` page's layout?
Is https://github.com/themefisher/bigspring-light/blob/master/layouts/_default/list.html the file I should duplicate?

**Reply by tfsomrat** [reply]
You can duplicate the `layouts/_default/` folder with list.html and single.html files. then change the folder name `_default` to `event`. and also duplicate the `content/blog` folder, and make it `content/event`. it will do the job. thank you
